getting-started/connectivity.md

```diff
@@ -27,6 +27,7 @@ Tremor has built-in support for metrics capture of data ingested and distributed
 
 Tremor supports a number of stable general purpose onramps:
 
+<<<<<<< HEAD
```

Comment:
Another one, I think we want to keep the relative path instead of the full one, since all pages are now the same